### DISCUSSION TITLE
Can I just replace manifest.json?

### DISCUSSION THREAD

**Discussion by darren365** [davewatson91/hass-amber-electric#3] [Q&A]
Thanks @davewatson91 for forking and creating a new manifest.json. If I just download this file and overwrite the existing one will that be suitable to preventing issues with HA related to it needing a version value?

**Comment by davewatson91** [comment]
@darren365 no worries - there was certainly self interest involved too. 

At this stage that's all I've updated, so replacing will work fine :)
